src/blocks.js

```diff
@@ -9193,9 +9193,24 @@ InputSlotMorph.prototype.menuFromDict = async function (
     }
 
     if (choices instanceof Function) {
-        choices = await choices.call(this);
+        if (!Process.prototype.enableJS) {
```

Comment:
Is it true for us that this point is only reached for JS extensions? Or can this point be reached in our code by a non-extension bit of code?